tabbycat/utils/tables.py

```diff
@@ -915,6 +915,24 @@ def add_debate_results_columns(self, debates):
                 else:
                     cell = self._result_cell_class_four(debateteam.points, cell)
 
+                if iron and (debateteam.iron > 0 or debateteam.iron_prev > 0):
+                    cell['text'] = "🗣️" + cell['text']
+
+                    popover_text = []
+                    if debateteam.iron > 0 and debateteam.iron_prev > 0:
+                        popover_text = _("Team iron-manned this round and the last.")
+                        warning_level = "text-info"
```

Comment:
Should the fact that they ironed last round take precedence (and thus get upgraded to `text-warning` level?

Reply:
A bit late, but for what it's worth, I concur with @philipbelesky.